src/modules/file/file.usecase.ts

```diff
@@ -246,8 +246,12 @@ export class FileUseCases {
   moveFilesToTrash(
     user: User,
     fileIds: FileAttributes['fileId'][],
-  ): Promise<void> {
-    return this.fileRepository.updateFilesStatusToTrashed(user, fileIds);
+    fileUuids: FileAttributes['uuid'][] = [],
+  ): Promise<[void, void]> {
+    return Promise.all([
```

Comment:
Hey, we do not want to move items that already trashed into the trash again, therefore, you need to add:
`where status = "EXISTS"`
So you can prevent concurrency issues if multiples devices are syncing.